### PR TITLE
EWL-4424 Adds link to body copy

### PR DESCRIPTION
**Pull request by designerneil**
Adds class to underline all links inside of p tags

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4424: SG2 | Create News Article Template](https://issues.ama-assn.org/browse/EWL-4424)

## Description

Adds link copy to body as requested.

## To Test

- [ ] http://localhost:3000/?p=pages-news
- [ ] Observe the link in the body on the first line
- [ ] Observe the margins between paragraphs
- [ ] Observe the margin on left of the article which matches the inline bio margin text

## Visual Regressions

none

## Relevant Screenshots/GIFs

screen shot 2018-01-30 at 10 46 17 am

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

### CODE REVIEW

**Review by labbydev — CHANGES_REQUESTED** [review]
@designerneil per the comments in the ticket this updates should also include adding back the margins between the paragraphs. Please add that as well and resolved the merge conflicts.

**Review by labbydev — CHANGES_REQUESTED** [review]
@designerneil this includes the margins on the paragraphs however, I think what UX was looking for was multiple shorter paragraphs to match the wireframe. Please review the paragraph spacing here (https://sketch.cloud/s/L0dLQ/p/news-article/article-desktop) and try to align the prototype more closely to the look of the paragraphs there.

**Comment by designerneil** [comment]
@labbydev I've made changes to the two column template so it can accommodate the margin in the article body so it matches the inline bio margin in the text. In addition, the paragraph now matches the mockup a bit more closely.

File: styleguide/source/_patterns/05-pages/news.twig [hunk 1]
```diff
@@ -9,6 +9,8 @@
 {% endblock %}
 
 {% block pageContent %}
+  {% include '@atoms/paragraph.twig' %}
+  {% include '@atoms/paragraph.twig' %}
```
Comment (labbydev) [inline styleguide/source/_patterns/05-pages/news.twig]: It would be more efficient to do this as a `foreach` so that we can add as many paragraphs as is necessary and we don't have to update the `news.twig` if we need to update that data in the `.json`

File: styleguide/source/assets/scss/01-atoms/_links.scss [hunk 2]
```diff
@@ -67,6 +67,11 @@ a {
   }
 }
 
+//Adds underlines to all links inside of p tags
+p a {
+  text-decoration: underline;
```
Comment (labbydev) [inline styleguide/source/assets/scss/01-atoms/_links.scss]: Are all links that don't have a specific class going to need this or only items that are inside `<p>` tags? I'm concerned that this could get buggy when dealing with WYSIWYG content or other content.

File: styleguide/source/_patterns/02-molecules/inline-bio.twig [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
-      "class": "ama__inline-bio__image"}
-    %}
+      {% include "@atoms/image.twig"  with {
+        "class": "ama__inline-bio__image"}
+      %}
   {% endif %}
 
-  {% set heading = bio.heading %}
-  {% include "@atoms/heading.twig" %}
+  <div class="ama__inline-bio__text">
```
Comment (labbydev) [inline styleguide/source/_patterns/02-molecules/inline-bio.twig:10]: what was the purpose of this change? Seems irrelevant to this ticket.

File: styleguide/source/_patterns/03-organisms/four-up-teaser.twig [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <div class="ama__four-up-teaser">
-  {% for item in fourUpTeaser.items[0:4] %}
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/four-up-teaser.twig:2]: This shouldn't be removed. It's an important note that the items never exceed 4 even when more than 4 are added.

File: styleguide/source/assets/scss/00-base/_typography.scss [hunk 5]
```diff
@@ -44,6 +44,12 @@ h5,
   @include font-size($h5-font-sizes);
 }
 
+p {
```
Comment (labbydev) [inline styleguide/source/assets/scss/00-base/_typography.scss:47]: Should these styles be on the `body`?

File: styleguide/source/assets/scss/01-atoms/_icons.scss [hunk 6]
```diff
@@ -43,7 +43,7 @@ $icons: (
 @each $name in $icons {
   .icon--#{$name} {
     display: inline-block;
-    background: url("../../assets/icons/svg/icon-" + $name + ".svg") no-repeat 0 0;
+    background: url("../icons/svg/icon-" + $name + ".svg") no-repeat 0 0;
```
Comment (labbydev) [inline styleguide/source/assets/scss/01-atoms/_icons.scss:46]: All of the icons are no longer appearing now. This pulls in a broken link.

File: styleguide/source/assets/scss/02-molecules/_tags.scss [hunk 7]
```diff
@@ -1,16 +1,23 @@
...
+    margin-right: 0;
   }
-}
 
-.ama__tag {
-  display: inline-block;
-  padding: ($gutter / 4) ($gutter / 2);
-  background: $gray-7;
-  color: $purple;
+  .ama__tag {
```
Comment (labbydev) [inline styleguide/source/assets/scss/02-molecules/_tags.scss]: This should probably not be nested within `ama__tags`

**Review by labbydev — COMMENTED** [review]
Please remove any irrelevant changes, you should be able to use `git checkout SHA -- path/to/filename` to remove any files that were changed unnecessarily.

File: styleguide/source/assets/scss/01-atoms/_links.scss (hunk 2 above)
Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_links.scss]: only links that are inside of p tags are going have underlines.

Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_links.scss]: let's revisit this when we implement the WYSIWYG  editors. It could get funky

File: styleguide/source/_patterns/02-molecules/inline-bio.twig (hunk 3 above)
Comment (designerneil) [reply styleguide/source/_patterns/02-molecules/inline-bio.twig:10]: this was done because the article body copy needs align with the bio text above it per UX comp.

File: styleguide/source/assets/scss/00-base/_typography.scss (hunk 5 above)
Comment (designerneil) [reply styleguide/source/assets/scss/00-base/_typography.scss:47]: body already has a style but the p tag had different sizing. This is to ensure all p are the same size no matter what.

File: styleguide/source/assets/scss/01-atoms/_icons.scss (hunk 6 above)
Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_icons.scss:46]: hmmm. this was done so in D8 the icon paths were not broken. The icons are not broken on my machine. Perhaps we can compare notes?

Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_icons.scss:46]: Did you try the gulp task gulp svg2twig?

File: styleguide/source/assets/scss/00-base/_typography.scss (hunk 5 above)
Comment (labbydev) [reply styleguide/source/assets/scss/00-base/_typography.scss:47]: I agree that all `<p>` should be the same I guess my question is why is the body not set to the default `<p>` sizing?

Comment (labbydev) [reply styleguide/source/assets/scss/00-base/_typography.scss:47]: If we need to change the size of the body copy of page that needs to happen on the default html element. All sizing of everything else should be relative to the base body copy. This kind of deviation is going to be a nightmare for maintaining the code moving forward.

Comment (designerneil) [reply styleguide/source/assets/scss/00-base/_typography.scss:47]: agreed making the appropriate changes now and removing the a tag inside of p.

**Review by labbydev — APPROVED** [review]
